sqlobject/src/main/java/org/jdbi/v3/sqlobject/BridgeMethodHandlerFactory.java

```diff
@@ -36,7 +37,30 @@
                 return IntStream.range(0, method.getParameterCount())
                     .allMatch(i -> methodParamTypes[i].isAssignableFrom(candidateParamTypes[i]));
             })
-            .<Handler>map(m -> (target, args, handle) -> m.invoke(target, args))
+            .<Handler>map(m -> {
+                final MethodHandle mh = unreflect(sqlObjectType, m);
```

Comment:
I think we could move `MethodHandle` inside the invoking lambda and just pass it around:
```java
(target, args, handle) -> {
    try {
        return unreflect(sqlObjectType, m).bindTo(target).invokeWithArguments(args);
    } catch (Throwable t) { // Handle et al should <X extends Throwable> not Exception!!!
         sneakyThrow(t);
         throw new AssertionError("unreachable", t);
    }
}
```

Reply:
I pulled it up so as to cache the lookup once up front.  I don't know how expensive it is, but it doesn't vary call to call...